lib/mixlib/shellout.rb

```diff
@@ -297,7 +289,6 @@ def parse_options(opts)
           self.password = setting
         when 'user'
           self.user = setting
-          self.with_logon = setting
```

Comment:
Yeah we could just add uid and gid to windows explicitly as a no-op with a  `# FIXME: delete these?` comment.